Source/Objects/movementobject.cpp

```diff
@@ -1402,11 +1402,24 @@ void MovementObject::CreateRiggedObject() {
     rigged_object_->char_id = GetID();
     palette.clear();
     rigged_object_->SetCharacterScriptGetter(character_script_getter);
+
     if (GetScriptParams()->HasParam("Character Scale")) {
         rigged_object_->SetCharScale(GetScriptParams()->ASGetFloat("Character Scale"));
+
+        float new_frust = GetScriptParams()->ASGetFloat("Character Scale") * 2;
+        if (new_frust <= 2.0f) {
+            kCullRadius = 2.0f;
+        } else if (new_frust >= 200.0f) {
+            kCullRadius = 200.0f;
+        } else {
+            kCullRadius = new_frust;
+        }
```

Comment:
This code is basically `kCullRadius = clamp(new_frust, 2.0f, 200.0f);`

Not sure if we have such a clamp implementation in our codebase already, but if we do, this would be a perfect place to use it.